app/views/distribution_mailer/reminder_email.html.erb

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>Hello <%= @partner.name %>,</p>
+    <p>This is a friendly reminder that you will be receiving a distribution tomorrow, <%= @distribution.issued_at.strftime("%A, %B #{@distribution.issued_at.day.ordinalize} %Y") %> </p>
```

Comment:
Apologies for not having this copy more specific in the issue description -- @seanmarcia or @benreyn might be good to weigh-in on this. I think the language might need to suggest that they will need to come to the Diaperbank to pick up the distribution. Even though the distributions are outgoing, they aren't delivered (AFAIK?)